requirements/constraints.txt

```diff
@@ -68,3 +68,13 @@ pennsieve>=999999999
 
 # User alternative e.g. parametrized fixture or request.getfixturevalue(.)
 pytest-lazy-fixture>=999999999
+
+# avoid downgrades of openapi-spec-validator related libraries
+referencing<=0.35.1
```

Comment:
do we still need that one? @pcrespov

Reply:
We use openapi-spec-validator. Not sure whether the latest version of this library still has this dependency
@GitHK did you check? If not, i propose doing that in a separate PR where we can review all these constraint files. This way we do not do more changes here.